_data/forms/config-admin.yml

```diff
@@ -3,12 +3,4 @@
 id: admin
 
 formGroups:
-  - preset: orgLevel
```

Comment:
Move components from admin form into a single preset to be use as the inline new admin in other forms.